src/components/storage/ReclaimSpaceModal.scss

```diff
@@ -13,8 +13,12 @@
     border-block-end: none;
   }
 
-  tr.reclaim-space-modal-table-row.reclaim-space-modal-device-level-0 {
-    border-block-start: var(--pf-v5-c-table--border-width--base) solid var(--pf-v5-c-table--BorderColor);
+  // Add a border between disk rows, except for the first one,
+  // since it already has a sticky header border that we shouldn't modify.
+  tbody:not(:nth-child(2)) {
+    tr.reclaim-space-modal-table-row.reclaim-space-modal-device-level-0 {
```

Comment:
This is an _extremely specific_ selector. Does it need to be? Can we make it much simpler yet still apply?

Reply:
Maybe we could drop ' tr.reclaim-space-modal-table-row.', I will check it when I get a bit more stable test suite, I am fighting with tests these days